content/docs/rad-cli-reference.rst

```diff
@@ -63,6 +64,7 @@ rad patch commands
                       <state2> ...'.
                       If the option '--fancy' is appended, the list is opened via
                       fzf (note that fzf has to be installed).
+                      Unread patches are marked with a * before the timestamp.
        propose      - Create a new patch from a single commit
```

Comment:
should be `after the timestamp`

Reply:
https://github.com/radicle-dev/radicle/pull/580